deps/openssl/config/archs/aix-gcc/opensslconf.h

```diff
@@ -235,7 +235,7 @@ extern "C" {
    even newer MIPS CPU's, but at the moment one size fits all for
    optimization options.  Older Sparc's work better with only UNROLL, but
    there's no way to tell at compile time what it is you're running on */
-
+ 
```

Comment:
OpenSSL Configure generates this whitespace but it is always removed with `git am --whitespace=fix`. So we can left it as it will be removed.